app/display/representation-javafx/src/main/java/org/csstudio/display/builder/representation/javafx/widgets/SymbolRepresentation.java

```diff
@@ -495,34 +496,47 @@ private void enableRunActionsOnMouseClick() {
         ColorAdjust increaseBrightness = new ColorAdjust(0, 0, 0.3, 0);
         ColorAdjust decreaseBrightness = new ColorAdjust(0, 0, -0.3, 0);
 
+        boolean[] buttonIsClicked = { false }; // Value is wrapped in an array as a workaround of the fact that Java doesn't allow non-final variables to be captured by closures.
```

Comment:
You could consider using AtomicBoolean instead of boolean array, and then use AtomicBoolean.get() and AtomicBoolean.set().